griptape/drivers/vector/pgvector_vector_store_driver.py

```diff
@@ -142,17 +146,17 @@ def query(
             vector = self.embedding_driver.embed_string(query)
 
             # The query should return both the vector and the distance metric score.
-            query = session.query(self._model, op(vector).label("score")).order_by(op(vector))
+            query_result = session.query(self._model, op(vector).label("score")).order_by(op(vector))  # pyright: ignore
```

Comment:
Is the `pyright: ignore` needed?

Reply:
Similar issue to the one I'm assuming is on the SQL Alchemy end... `Object of type "None" cannot be called` which I think is pointing at `self.engine` possibly being `None`